pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java

```diff
@@ -209,9 +261,16 @@ public PartitionedTopicMetadata getPartitionedMetadata(@PathParam("tenant") Stri
         @ApiResponse(code = 404, message = "Partitioned topic does not exist"),
         @ApiResponse(code = 412, message = "Partitioned topic name is invalid")
     })
-    public void deletePartitionedTopic(@PathParam("tenant") String tenant, @PathParam("namespace") String namespace,
+    public void deletePartitionedTopic(
+            @ApiParam(value = "Specify the tenant to which this topic belongs", required = true)
+            @PathParam("tenant") String tenant,
+            @ApiParam(value = "Specify the namespace to which this topic belongs", required = true)
+            @PathParam("namespace") String namespace,
+            @ApiParam(value = "Specifies topic name", required = true)
             @PathParam("topic") @Encoded String encodedTopic,
+            @ApiParam(value = "Close all producer/consumer/replicator and delete topic forcefully", defaultValue = "false", type = "boolean")
```

Comment:
Could we use "stop" or "terminate" to replace "close"? 
If so, you can check and refine the same cases in this file.